src/SubmitButton/SubmitButton.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Styles.css'
+
+function SubmitButton(props) {
+    return(
+        <button className='defaultButton'>{props.label}</button>
```

Comment:
Please change name of the class so it's consistent with name of the component (but start with small key)

Reply:
Corrected